app/templates/screwdriver.yaml

```diff
@@ -0,0 +1,6 @@
+jobs:
+  main:
+    image: node:6
+    steps:
+      - init: npm install
+      - test: npm test
```

Comment:
Agreed. There's something wrong with the generator pipeline, in general. It doesn't seem to detect this PR.